packages/visualizations/src/components/MapPoi/Map.ts

```diff
@@ -71,6 +77,30 @@ export default class MapPOI {
         this.mapResizeObserver.observe(container);
     }
 
+    /**
+     * How cursor should react
+     * - drag: move
+     * - hover a feature with a popup: pointer
+     */
+    private initializeCursorBehavior(map: maplibregl.Map) {
+        const canvas = map.getCanvas();
+        map.on('dragstart', () => {
+            canvas.style.cursor = CURSOR.DRAG;
+        });
+        map.on('dragend', () => {
+            canvas.style.cursor = CURSOR.DEFAULT;
+        });
+        map.on('mousemove', ({ point }) => {
+            const features = map.queryRenderedFeatures(point, { layers: this.layerIds });
+            const isMovingOverFeatureWithPopup =
+                features.length &&
+                features.some((feature) =>
+                    Object.keys(this.popupsConfiguration).includes(feature.layer.id)
+                );
+            canvas.style.cursor = isMovingOverFeatureWithPopup ? CURSOR.HOVER : CURSOR.DEFAULT;
```

Comment:
The "hover" state should probably only apply when map is `interactive: true` as opening a popup is blocked when `interactive: false`.

Reply:
Should be fixed now (https://github.com/opendatasoft/ods-dataviz-sdk/pull/191/commits/51673d0a685aec53ec8270f8d203064053aa2caa)